Documentation/CRDs/Cluster/ceph-cluster-crd.md

```diff
@@ -45,7 +45,9 @@ If this value is empty, each pod will get an ephemeral directory to store their
     * `port`: Allows to change the default port where the dashboard is served
     * `ssl`: Whether to serve the dashboard via SSL, ignored on Ceph versions older than `13.2.2`
 * `monitoring`: Settings for monitoring Ceph using Prometheus. To enable monitoring on your cluster see the [monitoring guide](../../Storage-Configuration/Monitoring/ceph-monitoring.md#prometheus-alerts).
-    * `enabled`: Whether to enable prometheus based monitoring for external or internal cluster
+    * `enabled`: Whether to enable prometheus based monitoring and alerts for an external or internal cluster. Default is false.
```

Comment:
Could be helpful for users to explain that this results in Prometheus ServiceMetric and Exports (IIRC) being created so they have a better understanding of `enabled` versus `metricsDisabled`.

Reply:
I'll push an attempt at more details...